app/js/main.js

```diff
@@ -328,17 +333,55 @@ function startup() {
 
   // Preview a captured frame
   frameReelRow.addEventListener("click", function(e) {
-    if (e.target.className === "frame-reel-img") {
-      if (winMode !== "playback") {
-        switchMode("playback");
+    if (e.target.className.includes("frame-reel-img")) {
```

Comment:
`classList.includes` is not a valid method. You are looking for `classList .contains`. Fixing this may affect behavior (since the code is actually valid), so be sure to test everything afterward.